conda-forge.yml

```
@@ -1,3 +1,6 @@
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```

Cross-compiled builds are running out of space on CI. This is a known issue (particularly with CUDA 11 builds). The typical solution is to more proactively clean up the CI image beforehand using [`azure`'s `free_disk_space`]( https://conda-forge.org/docs/maintainer/conda_forge_yml/#azure ). Have included this suggestion below

```suggestion
azure:
  free_disk_space: true
build_platform:
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
```

Note: This will also need a re-render to take effect